src/index.html

```diff
@@ -7,8 +7,53 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+      <div class="container">
+        <div class="header-body">
+          <a href="#home" class="logo-link">
+            <img
+              src="./images/logo.png"
+              alt="logo"
+              class="logo"
+            >
+          </a>
+
+          <nav class="nav">
+            <ul class="menu-list">
+              <li class="menu-item">
+                <a class="menu-link is-active" href="#">Apple</a>
+              </li>
+              <li class="menu-item">
+                <a class="menu-link" href="#">Samsung</a>
+              </li>
+              <li class="menu-item">
+                <a class="menu-link" href="#">Smartphones</a>
+              </li>
+              <li class="menu-item"
+                  data-qa="hover"
+              >
```

Comment:
Just write in one line if you have two or less attributes 
```suggestion
              <li class="menu-item" data-qa="hover">
```